src/boot/ostree-relabel-etc.service

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2013 Colin Walters <walters@verbum.org>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library. If not, see <https://www.gnu.org/licenses/>.
+
+[Unit]
+Description=OSTree Relabel etc
+Documentation=man:ostree(1)
+DefaultDependencies=no
+ConditionKernelCommandLine=ostree
+OnFailure=emergency.target
+Conflicts=umount.target
+# Run after core mounts
+After=-.mount var.mount
+After=systemd-remount-fs.service
+# But we run *before* most other core bootup services that need write access to /etc and /var
+Before=local-fs.target umount.target
+Before=systemd-random-seed.service plymouth-read-write.service systemd-journal-flush.service systemd-sysusers.service
+Before=systemd-tmpfiles-setup.service systemd-rfkill.service systemd-rfkill.socket
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/lib/ostree/ostree-relabel-etc
```

Comment:
Hmm, so far we haven't introduced many new little binaries, but instead we've been tending to dispatch via hidden CLI verbs, see e.g. `ostree admin finalize-staged`. I'd slightly prefer that, i.e. we introduce a new `ostree admin relabel-etc` (which should barf if it's not run under systemd, i.e. we check `INVOCATION_ID` etc).